src/native/libs/CMakeLists.txt

```diff
@@ -144,6 +144,7 @@ if (CLR_CMAKE_TARGET_UNIX OR CLR_CMAKE_TARGET_BROWSER)
         add_compile_options(-Wno-typedef-redefinition)
         add_compile_options(-Wno-c11-extensions)
         add_compile_options(-Wno-thread-safety-analysis)
+        add_compile_options(-Wno-strict-prototypes)
```

Comment:
Just curious, why was this needed?

Reply:
The compiler is more strict and there are many new warnings. I opened https://github.com/dotnet/runtime/issues/69142 so that we can fix it in another PR.